src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp

```diff
@@ -562,6 +565,9 @@ namespace
 
         virtual void init(HWND hwnd) override
         {
+            m_radio_render_only = GetDlgItem(hwnd, IDC_RADIO_RENDER);
```

Comment:
We should always strive to keep orderings consistent, everywhere. Here, ordering of member declarations and ordering of initializations should match.